api/src/main/resources/marquez/db/migration/V24__alter_jobs.sql

```diff
@@ -0,0 +1,20 @@
+ALTER TABLE jobs ADD job_context_template_uuid UUID;
+ALTER TABLE jobs ADD location_template varchar;
+ALTER TABLE jobs ADD inputs_template JSONB;
+ALTER TABLE jobs ADD outputs_template JSONB;
```

Comment:
It think want to prefix the columns with `current_*`. For example:

```
current_job_context_uuid
current_ location
etc
```

Reply:
Agreed. Maybe for column names / when we modify the data model we can open an issue to quickly decide on naming beforehand?

Reply:
Also, my fault for providing late feedback 😬